packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -123,6 +125,11 @@ class FlexibleSpaceBar extends StatefulWidget {
   /// not centered, `EdgeInsetsDirectional.only(start: 0, bottom: 16)` otherwise.
   final EdgeInsetsGeometry? titlePadding;
 
+  /// Defines how much the title is scaled when expanded.
```

Comment:
It would be helpful to remind the reader what causes the title to be scaled (it's when the FlexibleSpaceBar expands because the user has scrolled downwards). It might also be worth explaining how the title is scaled: in both x and y and without moving the center-X and bottom-Y of the title.